package.json

```diff
@@ -5,13 +5,22 @@
   "main": "build/index.js",
   "peerDependencies": {
     "prop-types": "^15.5.8",
-    "react": "^15.3.0",
+    "react": "^15.3.0 || ^16.0.0",
```

Comment:
set `peerDependencies` :  "^15.x.x || ^16.x.x"

can be apply to : `prop-types` and `tinycolor2`